mvc/views/view-components.rst

```diff
@@ -21,54 +30,98 @@ New to ASP.NET MVC 6, view components are similar to partial views, but they are
-- If the user is in the admin role, an admin panel is rendered.
+- If the user is not logged in, a login panel is rendered
+- If the user is logged in, links to log out and manage account are rendered
+- If the user is in the admin role, an admin panel is rendered
+
+The view component ``User`` property gives you access to a user's claims, so you can render data depending on the user's claims. You can add this view component view to the layout page and have it render user-specific data throughout the whole application. View components don’t use model binding, and only depend on the data you provide when calling into it. 
+
+A view component consists of two parts, the class (typically derived from  ``ViewComponent``) and the Razor view which calls methods in the view component class). Like controllers, a view component can be a POCO, but most developers will want to take advantage of the methods and properties available by deriving from ``ViewComponent``.
 
-You can also create a view component that gets and renders data depending on the user's claims. You can add this view component view to the layout page and have it get and render user-specific data throughout the whole application. View components don’t use model binding, and only depend on the data you provide when calling into it. 
+See `ViewComponent properties <https://docs.asp.net/projects/api/en/latest/autoapi/Microsoft/AspNet/Mvc/ViewComponent/index.html#properties>`__ and `ViewComponent Class <https://docs.asp.net/projects/api/en/latest/autoapi/Microsoft/AspNet/Mvc/ViewComponent/index.html>`__.
 
-A view component consists of two parts, the class (typically derived from  ``ViewComponent``) and the Razor view which calls methods in the view component class. Like controllers, a view component can be a POCO, but most users will want to take advantage of the methods and properties available by deriving from ``ViewComponent``.
+Creating a view component
+---------------------------
+
+This section contains the high level requirements to create a view component. Later in the article we'll examine each step in detail and create a view component.
+
+The view component class
+^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 A view component class can be created by any of the following:
 
-- Deriving from `ViewComponent`.
-- Decorating the class with the ``[ViewComponent]`` attribute, or deriving from a class with the ``[ViewComponent]`` attribute.
-- Creating a class where the name ends with the suffix *ViewComponent*.
+- Deriving from `ViewComponent`
+- Decorating a class with the ``[ViewComponent]`` attribute, or deriving from a class with the ``[ViewComponent]`` attribute
+- Creating a class where the name ends with the suffix *ViewComponent*
 
-Like controllers, view components must be public, non-nested, non-abstract classes.
+Like controllers, view components must be public, non-nested, and non-abstract classes.
 
-Examine the ViewComponent class
---------------------------------
+View component methods
```

Comment:
I think this section would read better as prose. Right now it reads like someones notes. How about something like "A view component defines its logic in an `InvokeAsync` method that returns an [IViewComponentResult](https://docs.asp.net/projects/api/en/latest/autoapi/Microsoft/AspNet/Mvc/IViewComponentResult/index.html?highlight=iviewcomponentresult). Alternatively you can define an `Invoke` method if your view component logic is synchronous. Parameters come directly from invocation of the view component, not from model binding. A view component never directly handles a request. Typically a view component initializes a model and passes it to a view by calling the [View](https://docs.asp.net/projects/api/en/latest/autoapi/Microsoft/AspNet/Mvc/ViewComponent/index.html?highlight=viewcomponent#meth-Microsoft.AspNet.Mvc.ViewComponent.View) method."